### ISSUE TITLE
Add more blocks to homepage

### ISSUE THREAD

**Issue by briggySmalls** [briggySmalls/skin-deep-server#138]
Anu's review:

> I like how the home page is split into sections. I would like to have 1 or 2 more sections on teh homepage. At the moment its Latest, Events and Videos, but I wonder if we can also have Skin Deep Meets and maybe one more? I think its great to have as many of teh categories seen on the homepage as possible without it being overwhelming, so that people can see more just from scrolling rahter than clicking through to a new page to get to the category.

**Comment by CiaranChris** [comment]
We just add more blocks? 😥

**Comment by briggySmalls** [comment]
Yeah, this is _really_ trivial to implement. In fact, I'll show you how. It's super easy.

This is only assigned to you because I thought I'd leave it to you to decide which blocks we want shown.
